### PR TITLE
Add option to open selected package repo with CTRL-O

### PR DESCRIPTION
**Pull request by Darukutsu**
It's good to use keyboard in terminal rather than mouse. It should be working, but just in case try it. I don't know "go" so...

### CODE REVIEW

File: internal/pacseek/display.go [hunk 1]
```diff
@@ -241,13 +241,14 @@ func (ps *UI) displayHelp() {
...
 		SetCellSimple(3, 0, "Shift+Left/Right: Change size of package list").
 		SetCellSimple(4, 0, "CTRL+S: Open/Close settings").
-		SetCellSimple(5, 0, "CTRL+N: Show these instructions").
-		SetCellSimple(6, 0, "CTRL+U: Perform sysupgrade").
-		SetCellSimple(7, 0, "CTRL+A: Perform AUR upgrade (if configured)").
-		SetCellSimple(8, 0, "CTRL+W: Wipe cache").
-		SetCellSimple(9, 0, "CTRL+P: Show PKGBUILD for selected package").
-		SetCellSimple(10, 0, "CTRL+G: Show list of upgradeable packages").
-		SetCellSimple(11, 0, "CTRL+L: Show list of all installed packages").
+		SetCellSimple(5, 0, "CTRL+O: Opens selected package repo url").
```
Comment (moson-mo) [inline internal/pacseek/display.go:244]: I would put that at position 10, next to CTRL+P and rephrase it to `CTRL+O: Show URL for selected package` to be in line with CTRL+P...

File: internal/pacseek/display.go [hunk 2]
```diff
@@ -241,13 +241,14 @@ func (ps *UI) displayHelp() {
...
+		SetCellSimple(6, 0, "CTRL+N: Show these instructions").
+		SetCellSimple(7, 0, "CTRL+U: Perform sysupgrade").
+		SetCellSimple(8, 0, "CTRL+A: Perform AUR upgrade (if configured)").
+		SetCellSimple(9, 0, "CTRL+W: Wipe cache").
+		SetCellSimple(10, 0, "CTRL+P: Show PKGBUILD for selected package").
+		SetCellSimple(11, 0, "CTRL+G: Show list of upgradeable packages").
+		SetCellSimple(12, 0, "CTRL+L: Show list of all installed packages").
 		SetCellSimple(13, 0, "CTRL+Q / ESC: Quit")
```
Comment (moson-mo) [inline internal/pacseek/display.go:252]: I'd prefer a gap between the "quit line" and the others. Should be at position 14...

File: internal/pacseek/setup.go [hunk 3]
```diff
@@ -263,6 +264,12 @@ func (ps *UI) setupKeyBindings() {
 			return nil
 		}
 
+		// CTRL+O - opens repo url
+		if event.Key() == tcell.KeyCtrlO {
+		  exec.Command("xdg-open", ps.selectedPackage.URL).Run();
```
Comment (moson-mo) [inline internal/pacseek/setup.go:269]: This segfaults when ps.selectedPackage is nil. There should be check in the if statement.

**Comment by Darukutsu** [comment]
I was also thinking about adding "sorting combinations" to `displayHelp()`as (personally) didn't know about them till checking man page.

File: internal/pacseek/setup.go [hunk 4]
```diff
@@ -243,6 +244,15 @@ func (ps *UI) setupKeyBindings() {
 			}
 		}
 
+		// CTRL+O - Show URL for selected package
+		if event.Key() == tcell.KeyCtrlO {
+			if ps.selectedPackage == nil {
```
Comment (moson-mo) [inline internal/pacseek/setup.go:249]: This can/should be done in one if statement. f.e.
```go
if event.Key() == tcell.KeyCtrlO && ps.selectedPackage != nil {
	exec.Command("xdg-open", ps.selectedPackage.URL).Run();
	return nil
}
```

**Comment by moson-mo** [comment]
> I was also thinking about adding "sorting combinations" to `displayHelp()`as (personally) didn't know about them till checking man page.

Imho, it's already too bloated on the usage screen.
I'll probably add note there to read the man page or wiki for further instructions.
